src/interceptors/XMLHttpRequest/XMLHttpRequestController.ts

```diff
@@ -561,7 +561,7 @@ export class XMLHttpRequestController {
        * @see https://xhr.spec.whatwg.org/#cross-origin-credentials
        */
       credentials: this.request.withCredentials ? 'include' : 'same-origin',
-      body: ['GET', 'HEAD'].includes(this.method)
+      body: ['GET', 'HEAD'].includes(this.method.toUpperCase())
```

Comment:
I've also added this fix to XMLHttpRequest. It's unlikely, but I say we better be fool-proof here, especially since this interceptor is for JSDOM mostly, where implementations may vary.